README.md

```diff
@@ -60,9 +60,10 @@ to track the versions of the following executable packages:
 package tools
 
 import (
+    _ "github.com/golang/protobuf/protoc-gen-go"
```

Comment:
If we make this documentation change, we should also change this to `google.golang.org/protobuf/cmd/protoc-gen-go`.

Reply:
Right! I just copied the instructions from [gRPC quickstart](https://grpc.io/docs/languages/go/quickstart/) but it should be updated.
I'll update the PR.